lib/support_ticket.rb

```diff
@@ -41,12 +41,6 @@ def payload
   end
 
   def send
-    self.class.client.ticket.create!({
-      subject: subject,
-      priority: PRIORITY,
-      requester: REQUESTER.merge(email: requester_email),
-      tags: TAGS,
-      comment: { body: body },
-    })
+    self.class.client.zendesk_client.tickets.create!(payload)
```

Comment:
this seems to be undoing a change from the `Fix deprecation notices` commit. Is it possible to remove the change from that commit?

Reply:
It's... not quite undoing it (you'll see that the end result is slightly different), but yeah, I might be able to tinker with the commits to go straight from the original version to this, let me see if I can do that.